.pipelines/templates/rp-int-deployment/template-push-aro-fluenbit-to-acr.yml

```diff
@@ -0,0 +1,15 @@
+parameters:
+  fluentbitImageTag: ""
+  fluentbitImage: ""
+steps:
+  - script: |
+      make publish-image-aro-multistage
```

Comment:
>  I see it's run in a previous step to deploy OCP images, but if there's a dependency, would it be better to make the login an explicit step instead of inserting it within a prior step?

I am separating it into a different template, and in a follow up I will try to add a dependency by 
> dependsOn: acr-login
  condition: failed()

where `condition` will receive the output of the `acr-login` template.